1-estrategia/README.md

```diff
@@ -1,23 +1,162 @@
+- Texto 
+- Audio 
+- Video 
+- Imagen 
+
+¿Que tipo de contenidos consumes acerca de la música? 
+- Noticia
+- Reportaje 
+- Entrevista 
+- Videoclip 
+- Audio
+- Crítica
+
+Valora del 0 al 5 el interés que tengas acerca de los siguientes géneros musicales: (0 no me interesa, 5 me interesa mucho)
+- Música clásica  
+- Rock 
+- Pop 
+- Reggaetón 
+- Jazz 
+- Trap 
+- Punk 
+- Rap 
+- Techno 
+- Flamenco 
+- Country 
+
+Si tienes interés en otro género musical escríbelo aquí:
+
+¿Tienes interés en saber cuales son las tendencias musicales de la actualidad? Sí/No
+
+¿Y las tendencias o nuevos temas del género/s que más escuchas? Sí/No
+
+¿Te interesa profundizar acerca de las canciones que escuchas? SÍ/No
+
+Si has respondido que sí, ¿en qué te interesa profundizar?
+- Autor  
+- Letra 
+- El por qué de la canción 
+- Significado 
+- Su repercusión 
+
+¿Cómo consigues información acerca de los conciertos/festivales que vaya a haber?
+- Internet
+- Boca a boca
+- Carteles publicitarios
+- Periódico 
+- Televisión
+- Radio
+
+¿Compras entradas por Internet? Sí/No
+
+Si has respondido que sí, ¿cómo lo haces?
+- Acudo directamente a webs de venta de entradas 
+- Escribo en mi navegador frases tipo: “entradas para concierto X”  
+- Acudo a la página web del artista/grupo  
+- Acudo a páginas web musicales que me informan sobre la venta de entradas 
+- Ninguna de las anteriores
+
+Una vez realizada la encuesta y analizadas las respuestas obtenidas, los resultados y las conclusiones que hemos sacado son las siguientes:
+
+**Tipo de formato y contenido para consumir información musical**
+
+Las primeras dos preguntas de la encuesta han sido sobre el tipo de formato y contenido que utilizan a la hora de consumir información sobre música. En el caso del tipo de formato, los resultados han sido muy parecidos. El porcentaje más alto (80% de los encuestados) y por tanto, el tipo de contenido que más consumen son el texto, el vídeo y la imagen. El audio ha obtenido un porcentaje bastante más bajo (20%), pero al obtener un porcentaje que no sea el 0, creemos que también podría ser una forma de insertar contenido en nuestra página web, aunque no se utilizará más que los demás formatos.
+
+En cuanto a los resultados del tipo de contenido que consumen en Internet, los porcentajes han sido más variados. Las entrevistas y los audios son los formatos que más se consumen por parte de los encuestados, ya que todas las personas que han rellenado la encuesta las han elegido. Por detrás de las entrevistas y los audios estarían las noticias con un porcentaje del 80%. Los videoclips y los reportajes también han sido elegidos pero con un porcentaje bastante menor. En última posición están las críticas con un 0% de porcentaje, por tanto, ninguno de los encuestados consume este tipo de contenido. Aun así, creemos que es conveniente optar por diferentes tipos de contenidos y por esta razón, no descartaremos ninguno de ellos, por el momento.
```

Comment:
No parece tener mucho sentido hacer una encuesta para ignorar sus resultados.

En cualquier caso, la principal duda con este apartado: ¿qué diferencia hay entre este _audio_ y el _audio_ del anterior apartado? ¿Qué diferencia le otorgáis vosotros? ¿Creéis que las personas que han respondido a la encuesta ven la misma diferencia, y por tanto han respondido en consonancia?